tests/test.py

```diff
@@ -1002,8 +1005,40 @@ def test_02_save_without_warning(self):
         self._mainmenu("Save")
         self._wait_saving()
         self._quit()  # save config
+        self._process().stdout.close()
+        self._process().wait(timeout=22)
         self.assertEqual(self._config()["preferences"]["show-save-warnings"], "False")
 
+    def test_03_retain_metadata(self):
+        """Merge metadata into first file and don't overwrite from info dict"""
+        self._start(["tests/test_metadata1.pdf", "tests/test_metadata2.pdf"])
+        self._mainmenu("Save")
+        self._save_as_chooser("output2.pdf", ["output2.pdf"])
+        self._wait_saving()
```

Comment:
_save_as_chooser already has _wait_saving

Reply:
Yes, thanks. This was a leftover from my desperate attempts to let the process terminate. Fixed.